### PR TITLE
9527 - added tabIndex and onKeyPress to rows in TableData.jsx to make…

### PR DESCRIPTION
**Pull request by brianpetway**
… the accessible by tabbing

### Author
- [ x] Story Book has been rebuilt (new build artifacts in `/docs`)
- [ x] Components have been rebuilt (new build artifacts in `/dist`)

👆These can be done by executing `npm run build` and committing the build artifacts.

- [ ] All new components have been exported in `/index.js` `(if applicable)`
- [ ] Design Review Complete
- [ ] Relevant Draft release notes have been updated describing these changes ([use this template](https://github.com/fedspendingtransparency/data-transparency-ui/blob/master/release_notes_template.md) to document your changes) `(if applicable)`
- [ ] Integration Status Updated on relevant documentation page `(if applicable)`
- [ ] `index.d.ts` updated with new prop types defined for new components. `(if applicable)`

### Reviewer
- [ ] Code review

### CODE REVIEW

File: components/table/TableData.jsx [hunk 1]
```diff
@@ -69,7 +69,9 @@ const TableData = ({
             return (
                 <tr
                     key={uniqueId()}
+                    tabIndex={0}
                     onClick={() => localClickHandler(row, i)}
+                    onKeyPress={() => localClickHandler(row, i)}
```
Comment (anjenkin) [inline components/table/TableData.jsx:74]: the drilldown should only work on tab + enter not just any keypress so there needs to be a check if it's the enter key.

Comment (brianpetway) [reply components/table/TableData.jsx]: Fixed.

**Review by anjenkin — CHANGES_REQUESTED** [review]
Desktop looks good.  On the mobile table, if you tab+enter, the text "View next level" appears, but the table cell loses focus.  You have to tab to the cell again and hit enter.  You could try to reset the focus to the clicked element after tab+enter.  Not sure if that will work reliably.

**Comment by brianpetway** [comment]
I did try to trap the focus on the row after the first 'enter' at mobile and couldn't find any method that worked.
I can spend more time on it this week if it's a priority.

**Comment by anjenkin** [comment]
Ok, that's fine to look at it later this week.

**Review by anjenkin — APPROVED** [review]

**Pull request merged**